lib/api/src/Cardano/Wallet/Api/Clients/Testnet/Id.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE DataKinds #-}
+
+module Cardano.Wallet.Api.Clients.Testnet.Id
+    (TN42)
+where
+
+import Cardano.Wallet.Primitive.NetworkId
+    ( NetworkDiscriminant (..)
+    )
+
+type TN42 = 'Testnet 42
```

Comment:
hehe, what about changing the type synonym name to something more "intuitive" and less surprising? For example `Testnet42` ?